packages/features/shell/Shell.tsx

```diff
@@ -847,14 +907,44 @@ function SideBar({ bannersHeight }: SideBarProps) {
 
         <div>
           <Tips />
-          <div data-testid="user-dropdown-trigger">
-            <span className="hidden lg:inline">
-              <UserDropdown />
-            </span>
-            <span className="hidden md:inline lg:hidden">
-              <UserDropdown small />
-            </span>
-          </div>
+          {bottomNavItems.map(({ icon: Icon, ...item }) => (
+            <Tooltip side="right" content={t(item.name)} className="lg:hidden" key={item.href}>
+              <Link
+                href={item.href}
+                aria-label={t(item.name)}
+                target={item.target}
+                className={classNames(
```

Comment:
To prevent the issue i mentioned above we have the `ButtonOrLink` component that changes the render based on if a href is passed